plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/vendored/VendoredIcebergVersionTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.vendored;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class VendoredIcebergVersionTest
+{
+    private static final Pattern icebergParquetVersionMatcher = Pattern.compile("org/apache/iceberg/iceberg-parquet/([^/]+)/");
+    private static String auditedIcebergVersion = "1.5.0";
+
+    @Test
+    public void checkIcebergVersion()
```

Comment:
@electrum you mentioned needing to audit iceberg for changes whenever it is updated. See this test to try to automate that reminder.

Reply:
Let's remove this since we're not really "vendoring" these classes. We own them now and they'll diverge as we rename and refactor the code.